extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -138,4 +176,15 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 		return notebooks;
 	}
+
+	getNavigation(uri: vscode.Uri): Thenable<azdata.nb.NavigationResult> {
+		let notebook = this._allNotebooks.get(uri.fsPath);
+		let result: azdata.nb.NavigationResult = {
+			hasNavigation: true,
+			previous: notebook.previousUri ? vscode.Uri.file(notebook.previousUri) : null,
```

Comment:
Needs guard against notebook being undefined.

Reply:
Added check for notebook!